__tests__/app.test.js

```diff
@@ -172,4 +172,31 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/articles", () => {
+    test("Status 200 - Responds with an array of articles of expected length", () => {
+      return request(app)
+        .get("/api/articles")
+        .then(({ body: { articles } }) => {
+          expect(articles).toHaveLength(12);
+        });
+    });
+    test("Status 200 - Articles have expected properties", () => {
+      return request(app)
+        .get("/api/articles")
+        .then(({ body: { articles } }) => {
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
```

Comment:
We don't need the body on each article here - common practice to get more detailed information when requesting a single resource